Tools/autotest/arducopter.py

```diff
@@ -11499,26 +11499,6 @@ def Clamp(self):
 
         self.reboot_sitl()  # because we set home
 
-    def GripperReleaseOnThrustLoss(self):
-        '''tests that gripper is released on thrust loss if option set'''
-
-        self.context_push()
```

Comment:
We can retain this test - all we need to do is `install_example_script_context` here for your new script